docker/suricata/dist/suricata.yaml

```diff
@@ -1762,15 +1766,15 @@ napatech:
     # (-1 = OFF, 1 - 100 = percentage of the host buffer that can be held back)
     # This may be enabled when sharing streams with another application.
     # Otherwise, it should be turned off.
-    hba: -1
+    #hba: -1
```

Comment:
Since Napatech is disabled by default, I ported over all the new default parameters.
This shouldn't be a problem unless you are using Napatech.

Reply:
Awesome!